src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        return new Ball(ColorSupplier.getRandomColor(), random.nextInt(100));
```

Comment:
All magic numbers in your code should be constants. How do you think what max number will be returned by random.nextInt(100);?

Reply:
!!?!?!?!?!?!?!??!?!?!!?!??!?!?!?!